components/gsl/Makefile

```diff
@@ -9,36 +9,39 @@
 #
 
 #
-# Copyright (c) 2013 Louis M. Picciano. All rights reserved.
+# Copyright (c) 2013-2016 Louis M. Picciano. All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include $(WS_TOP)/make-rules/shared-macros.mk
```

Comment:
but it has been merged https://github.com/OpenIndiana/oi-userland/blob/8b5b0c6c5686e7f219f920eefd19e6b175db49f4/components/gsl/Makefile#L15